third_party/txt/src/txt/font_collection.cc

```diff
@@ -198,6 +200,7 @@ std::shared_ptr<minikin::FontFamily> FontCollection::FindFontFamilyInManagers(
 std::shared_ptr<minikin::FontFamily> FontCollection::CreateMinikinFontFamily(
     const sk_sp<SkFontMgr>& manager,
     const std::string& family_name) {
+  TRACE_EVENT0("flutter", "FontCollection::CreateMinikinFontFamily");
```

Comment:
Use the varargs variant `FML_TRACE_EVENT` to log the family name as well.

Reply:
How about the `TRACE_EVENT1` in the new patch?